backend/api/parse_pdf.py

```
@@ -520,6 +520,73 @@ async def get_default_personas():
+        
+        # Upload to temporary location
+        file_obj = BytesIO(file_contents)
+        temp_url = await wasabi_service.upload_file(file_obj, temp_s3_key, file_type)
+        
+        if temp_url:
+            metadata["temp_pdf_url"] = temp_url
+            debug_log(f"[PDF_METADATA] Uploaded large file to temporary storage: {temp_url}")
+        else:
+            # Fallback: set flag if upload failed
+            metadata["needs_upload"] = True
+            debug_log(f"[PDF_METADATA] Failed to upload large file, setting needs_upload flag")
```

Ruff (0.14.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)